src/modules/appFunction.js

```diff
@@ -55,6 +59,19 @@ const toggleIcon = (btn) => {
   });
 };
 
+export const removeAllCompleted = () => {
+  tasksArray.forEach((task) => {
+    if (task.completed) {
+      const taskIndex = tasksArray.indexOf(task);
+      tasksArray.splice(taskIndex, 1);
+    }
+  });
```

Comment:
- [ ] as part of the requirements, kindly use the `filter()` method to clear all the completed tasks 🙏